src/components/SpeakerCard/SpeakerCard.scss

```diff
@@ -0,0 +1,78 @@
+@use "~stylesheets/typography";
+
+.SpeakerCard {
+  padding: 4% 5%;
+  margin: auto;
+
+  &__firstLayer {
+    width: 18em;
+    background: #ffd9e0;
+    border-radius: 72px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__secondLayer {
+    background: #fdfdee;
+    box-shadow: 0 0 20px 11px rgba(89, 93, 130, 0.26);
+    border-radius: 72px;
+    height: 20em;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__thirdLayer {
+    width: 100%;
+    background: #fdfdee;
+    box-shadow: 0 0 20px 11px rgba(89, 93, 130, 0.17);
+    border-radius: 72px;
+    height: 17em;
+  }
+
+  &__infoContainer {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 5% 0;
+  }
+
+  &__info {
+    @include typography.speaker-card-info;
+    margin: 0.1em 0;
+    text-decoration: none;
+  }
+
+  &__divider {
+    border-top: 2px solid #898989;
+    width: 75%;
+  }
+
+  &__nameContainer {
+    margin-top: 5%;
+  }
+
+  &__name {
+    @include typography.speaker-card-name;
+  }
+
+  &__imageContainer {
+    display: flex;
+    justify-content: center;
+    height: 100%;
+  }
+
+  &__image {
+    border-radius: 72px;
+    width: 90%;
+  }
```

Comment:
See if you can add y-axis padding to help space the image within the third container a bit better.

Reply:
https://github.com/CruzHacks/cruzhacks-2021-website/pull/142#discussion_r550977362